canopy/app/src/App.js

```diff
@@ -66,18 +82,22 @@ function App() {
       ) {
         appSapling.current(saplingDomNode.current);
       }
-    })();
+    })().catch(e => {
+      throw Error(e);
+    });
   }, []);
 
-  const userSaplingRoutes = userSaplingManifests.map(
-    ({ displayName, namespace, icon }) => {
+  const userSaplingRoutes = userSaplingManifests
+    .filter(sapling => {
+      return sapling.namespace !== 'profile';
```

Comment:
If we use the requiredUserSaplings convention, this should probably look like
```
    .filter(sapling => {
      !requiredUserSaplings.includes(sapling.namespace)
    })
```

but see that other comment.